ehr/resources/queries/ehr_lookups/procedures.query.xml

```diff
@@ -24,6 +24,16 @@
                             <fkColumnName>value</fkColumnName>
                         </fk>
                     </column>
+                    <!-- Kollil, 12/08/2022: Added a new column (drop down list) to display the USDA pain categories. -->
+                    <column columnName="painCategories">
+                        <columnTitle>Pain Categories</columnTitle>
+                        <fk>
+                            <fkDbSchema>ehr_lookups</fkDbSchema>
+                            <fkTable>Procedure_Pain_Categories</fkTable>
+                            <fkColumnName>rowid</fkColumnName>
```

Comment:
@kollil  We generally try not to use the rowId for lookup columns. Is there a reason you can't use the value?

Reply:
Sure. I have no particular reason to do that way. I saw similar code for a different field and followed the same thing. I can change it to "value".

Reply:
Sorry formatting got kind of messed up on that